text-ui-test/input.txt

```diff
@@ -0,0 +1,11 @@
+todo read book
+list
+deadline return book /by Sunday
+list
+event project meeting /from Mon 2pm /to 4pm
```

Comment:
consider adding  "wrong user input" to test whether the application breaks if e.g. deadline is used without /by.